content/en/events/banners/ambient-release.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Ambient Alpha Release
+period_start: latest_release
+period_duration: 30
+max_impressions: 4
+link: https://istio.io/latest/news/releases/1.18.x/announcing-1.18/#ambient-mesh
```

Comment:
```suggestion
link: https://istio.io/docs/ops/ambient/getting-started/
```

Think we should link to the getting started doc instead since the banner is a recommendation to try it out and we specify a minor version in the banner to indicate what minor version they should try.

Reply:
Agreed. I think I can merge a suggestion as a codeowner.

Reply:
@craigbox - Do you want me to go ahead and commit that suggestion?